lib/charms/layer/tls_client.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2016-2017 Canonical Ltd.
+#
+# This file is part of the Snap layer for Juju.
```

Comment:
I missed this before, sorry. Can you remove this line or update it to refer to tls-client? I'd let it slide but, y'know, legal text.